### ISSUE TITLE
Github: discoveryDocument not defined

### ISSUE THREAD

**Issue by SevereOverfl0w** [Widen/cloudfront-auth#1]
https://github.com/Widen/cloudfront-auth/blob/e6245a63ea7667ed7911b39ea5a478551a5ddd99/authn/github.index.js#L128

I can't see where this is defined. This makes Github unusable as auth provider.

**Comment by SevereOverfl0w** [comment]
```
diff --git a/authn/github.index.js b/authn/github.index.js
index 87e4b3e..d3beea2 100644
--- a/authn/github.index.js
+++ b/authn/github.index.js
@@ -125,7 +125,7 @@ function redirect(request, headers, callback) {
     headers: {
       "location" : [{
         "key": "Location",
-        "value": discoveryDocument.authorization_endpoint + '?' + querystring
+        "value": config.AUTHORIZATION_ENDPOINT + '?' + querystring
       }],
       "set-cookie" : [{
         "key": "Set-Cookie",
```

This patch fixed it for me. I won't claim it has no adverse effects though.

**Comment by paytongarland** [comment]
Good catch on this. That is exactly right. I had made some recent big changes to the project structure and didn't remove the discovery document reference from the github index file (belongs in openid index file).

If you would like to create a PR with your changes, I'm more than happy to accept them. Otherwise I can get those changes made soon.

**Comment by paytongarland** [comment]
Issue resolved in e97d679369cec0c59aaa1853380a466100e41b3e

Thanks to @SevereOverfl0w for finding this.
